lib/external-housing-available/external-housing-available.controllers.js

```diff
@@ -192,6 +190,31 @@ module.exports = (models) => {
       return h
         .response(returnAllExternalHousingAvailable);
     },
+    getOneExternalHousingAvailable: async function(request, h) {
+      let returnOneExternalHousingAvailable;
+      try {
+        returnOneExternalHousingAvailable = await getOneExternalHousingAvailable(request.params.id);
+      } catch (e) {
+        throw Boom.badImplementation(`Error during getOneExternalHousingAvailable(). ${e}`);
+      }
+      return h
+        .response(returnOneExternalHousingAvailable);
+    },
+    deleteExternalHousingAvailable: function(request, h) {
+      if (request.auth.credentials.subjectId !== process.env.SUPER_ADMIN) {
+        throw Boom.forbidden('Must be an admin account to delete an external housing available');
+      }
+      return models.ExternalHousingAvailable.destroy({
+        where: {
+          id: request.params.id,
+        },
+      })
+      .then(response => {
+        return h
+        .response(response)
+        .code(202);
+      });
```

Comment:
I can see this is from Properties. If I was writing this again today, I'd probably write it a little differently, but it's only a matter of style. 

This is all fine, but if you are curious, the way I'd write it now would be similar other methods where:

we initialize an instance variable, 
wrap the db call in a try catch and assign the result to the instance variable. 
put any implementation errors in the catch. 
and then we'd return the instanceVariable after (and outside) the try catch in our h.response with the 202 code.

Reply:
I'd also add a pre handler to the route to handle the ensureAdmin, and check if propertyExists, but that's even more advanced and would require some larger additions to the codebase, so definitely don't worry about that. but if you'd sometime I can show you how that's done as that's where the codebase is probably heading.